### DISCUSSION TITLE
I would love to contribute my knowledge to the wiki section. 🤔 with a team!

### DISCUSSION THREAD

**Discussion by berlintay** [microsoft/terminal#16238] [General]
# I saw the [Microsoft/terminal/Wiki ](https://github.com/microsoft/terminal/wiki),  it needs some ***Life*** !
 and It looks like it needs some tweaking.
 are there any present projects to contribute? as a team?

if not, we should talk about this!!!

Looking Sad  :-( 

![image](https://github.com/microsoft/terminal/assets/128968711/2b9726d9-8d90-4e0e-a704-86a678849db5)

**Comment by zadjii-msft** [comment]
We don't really use the wiki for anything except for certain team notes, debugging tips, and info on certain experimental features. Almost all of our docs really live on https://learn.microsoft.com/en-us/windows/terminal/

What kind of thing were you thinking of contributing to the wiki?
